src/components/FilterList/FilterList.jsx

```diff
@@ -3,6 +3,81 @@ const FilterListItem = require('../FilterListItem')
 const hookUpStickyNav = require('../hookUpStickyNav')
 
 class FilterList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { selectedIndex: null }
+    this.onKeyUp = this.onKeyUp.bind(this)
+    this.onKeyDown = this.onKeyDown.bind(this)
+  }
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.onKeyUp)
+    document.addEventListener('keydown', this.onKeyDown)
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.onKeyUp)
+    document.removeEventListener('keydown', this.onKeyDown)
+  }
+
+  onKeyUp(event) {
+    if (event.key === 'ArrowUp') {
+      this.focusPreviousFilter()
+    } else if (event.key === 'ArrowDown') {
+      this.focusNextFilter()
+    } else if (event.key === 'Escape') {
+      this.setState({ selectedIndex: null })
+    } else if (typeof this.state.selectedIndex === 'number') {
+      if (event.key === 'Enter') {
+        this.editFocusedFilter()
+      } else if (event.key === 'Backspace') {
+        this.deleteFocusedFilter()
+      }
+    }
+  }
```

Comment:
These shortcuts are perfectly fine for now. At some point I really want to use gmail's shortcuts:

- `j` - down
- `k` - up
- `e` - archive
- `z` - undo